AnkiDroid/src/main/java/com/ichi2/anki/analytics/UsageAnalytics.java

```diff
@@ -278,4 +285,46 @@ public static void sendAnalyticsException(@NonNull String description, boolean f
         }
         sAnalytics.exception().exceptionDescription(description).exceptionFatal(fatal).sendAsync();
     }
+
+
+    /**
+     * An Android-specific device config generator. Without this it's "Desktop" and unknown for all hardware.
+     * It is interesting to us what devices people use though (for instance: is Amazon Kindle support worth it?
+     * Is anyone still using e-ink devices? How many people are on tablets? ChromeOS?)
+     */
+    private static class AndroidDefaultRequest extends DefaultRequest {
+        private DefaultRequest setAndroidRequestParameters(Context context) {
+
+            // Are we running on really large screens or small screens or ?
+            try {
+                WindowManager wm = (WindowManager) context.getSystemService(Context.WINDOW_SERVICE);
+                Display display = wm.getDefaultDisplay();
+                Point size = new Point();
+                display.getSize(size);
+                this.screenResolution(size.x + "x" + size.y);
+            } catch (RuntimeException e) {
+                // nothing much to do here, it means we couldn't get WindowManager
+            }
+
+            // We can have up to 20 of these - there might be other things we want to know
+            // but simply seeing what hardware we are running on should be useful
+            this.customDimension(1, Build.VERSION.RELEASE); // systemVersion
```

Comment:
Got this confused with `BuildConfig.DEBUG`
```suggestion
            this.customDimension(1, Build.VERSION.RELEASE); // readable version: "2.12alpha9"
```

Reply:
No, VERSION.RELEASE is systemVersion, like "10" for Android 10. After maintaining react-native-device-info for 2 years, I know my build constants ;-)

Reply:
🤦 Definitely needed those comments then, thanks!